CMakeLists.txt

```diff
@@ -313,19 +313,17 @@ ELSE()
 	MESSAGE(WARNING "Make sure you use a version compatible with the chosen SDL version.")
 ENDIF()
 
-IF((NOT OPENGL_BACKEND STREQUAL "None") OR USE_OPENGL)
```

Comment:
It would be better to leave the original check  (without USE_OPENGL), so old configs work too without extra code. Or add a compatibility if check to force None to 0 and everything else to 1.

Reply:
Just to be clear, by _old config_ do you refer to anything else but a fresh generator-run by CMake? I just try to image to whom this is a concern.

Reply:
Packagers and anyone else that may be passing it directly.

Reply:
Ok, then I test against _not "None"_ again.

Reply:
@lynxlynxlynx Please look again.